.gitignore

```diff
@@ -6,6 +6,8 @@ __pycache__/
 # C extensions
 *.so
 *.vscode
+**/migrations/*
```

Comment:
It's fine to commit migrations to version control, you don't need these

Reply:
Okay have worked on that